src/bus_monitor.cpp

```diff
@@ -217,7 +217,7 @@ void PELListener::PELEventCallBack(sdbusplus::message::message& msg)
                         types::Byte byte =
                             ::strtoul(valueAtIndexZero, nullptr, 16);
 
-                        if ((byte & constants::terminatingBit) != 0x00 &&
+                        if ((byte & constants::terminatingBits) != 0x00 &&
```

Comment:
As per the spec, 0th bit and 2nd bit from MSB end will account for check stop and terminating bit respectively.
Performing an AND with 0xC in this case will not detect terminating bit.